create/index.md

```diff
@@ -0,0 +1,93 @@
+---
+title: MonoGame Starter Kits
```

Comment:
I don't think "Create" is exclusively starter kits.

It is starter kits, demos with source, link to MG resources, etc.